lang/languages.xml

```diff
@@ -31,7 +31,7 @@ List of available languages for your wallet's seed:
         <language display_name="日本語" locale="ja_JP" wallet_language="日本語" flag="/lang/flags/japan.png" qs="none"/>
         <language display_name="Nederlands" locale="nl_NL" wallet_language="Nederlands" flag="/lang/flags/netherlands.png" qs="none"/>
         <!-- <language display_name="Polski" locale="pl_PL" wallet_language="English" flag="/lang/flags/poland.png" qs="none"/> -->
-        <language display_name="Português (BR)" locale="pt-br_BR" wallet_language="Português" flag="/lang/flags/brazil.png" qs="none"/>
+        <language display_name="Português (BR)" locale="pt-br_BR" wallet_language="Português (BR)" flag="/lang/flags/brazil.png" qs="none"/>
```

Comment:
Please leave this string as it is or it won't be correctly processed

Reply:
Ok!